nautobot/ipam/views.py

```diff
@@ -498,19 +483,22 @@ class PrefixPrefixesView(generic.ObjectView):
 
     def get_extra_context(self, request, instance):
         # Child prefixes table
-        child_prefixes = (
-            instance.descendants()
-            .restrict(request.user, "view")
-            .select_related("parent", "status", "role", "vlan", "namespace")
-            .annotate(location_count=count_related(Location, "prefixes"))
-        )
+        child_prefixes = instance.descendants().restrict(request.user, "view")
 
         # Add available prefixes to the table if requested
         if child_prefixes and request.GET.get("show_available", "true") == "true":
-            child_prefixes = add_available_prefixes(instance.prefix, child_prefixes)
 
-        prefix_table = tables.PrefixDetailTable(child_prefixes, hide_hierarchy_ui=True)
-        prefix_table.exclude = ("namespace",)
+            def data_transform_callback(prefixes):
+                return add_available_prefixes(instance.prefix, prefixes)
```

Comment:
Just wondering, what looks better: current function def or: `data_transform_callback = lambda prefixes: add_available_prefixes(instance.prefix, prefixes)`

Reply:
I actually went with the lambda initially but pylint yelled at me. :-)